core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -91,6 +91,44 @@ default ConnectorTableHandle getTableHandleForStatisticsCollection(ConnectorSess
         throw new TrinoException(NOT_SUPPORTED, "This connector does not support analyze");
     }
 
+    /**
+     * Create initial handle for execution of table procedure. The handle will be used through planning process. It will be converted to final
+     * handle used for execution via @{link {@link ConnectorMetadata#}beginTableExecute}
+     */
+    @Nullable
```

Comment:
remove

Reply:
It should stay here. We are mapping null to "table does not exist error". It works the same for `getTableHandle`